subworkflows/nf-neuro/tractoflow/nextflow.config

```diff
@@ -0,0 +1,107 @@
+    b0_max_threshold                                = 10
+    bvalue_tolerance                                = 20
+
+    //**SH fitting**//
+    dwi_signal_sh_fit                               = true
+    dwi_signal_sh_fit_order                         = 6
+    dwi_signal_sh_fit_basis                         = "descoteaux07"
+    dwi_signal_sh_fit_shell                         = null
+
+    //**Denoise dwi (dwidenoise in Mrtrix3)**//
+    dwi_denoise                                     = true
+    dwi_denoise_patch_size                          = 7
+
+    //**GIBBS CORRECTION (mrdegibbs in Mrtrix3)**//
+    dwi_degibbs                                     = false
+
+    //**Topup**//
+    dwi_topup                                       = true
+    dwi_topup_config_file                           = "b02b0.cnf"
+
+    //**Eddy**//
+    dwi_eddy                                        = true
+    dwi_eddy_executable                             = "eddy_cpu"
+    dwi_eddy_restore_slices                         = true
+
+    //**Denoise T1**//
+    t1_denoise                                      = true
+
+    //**Resample T1**//
+    t1_resample                                     = true
+    t1_resample_resolution_mm_iso                   = 1
+    t1_resample_interpolation                       = "lin"
+
+    //**Normalize DWI**//
+    dwi_n4_density                                  = 1
+    dwi_n4_subsampling                              = 2
+
+    //**Resample DWI**//
+    dwi_resample                                    = true
+    dwi_resample_resolution_mm_iso                  = 1
+    dwi_resample_interpolation                      = "lin"
+
+    //**Extract DTI shells using this value as maximum**//
+    dti_max_bvalue                                  = 1200
+    dti_shells_to_fit                               = null
+
+    //**Extract fODF shells using this value as minimum**//
+    fodf_min_bvalue                                 = 700
+    fodf_shells_to_fit                              = null
+
+    //**Compute fiber response function (frf)**//
+    frf_fa_max_threshold                            = 0.7
+    frf_fa_min_threshold                            = 0.5
+    frf_min_n_voxels                                = 300
+    frf_roi_radius                                  = 20
+    frf_value_to_force                              = null
+
+    //**Mean fiber response function (frf)**//
+    frf_average_from_data                           = false
+
+    //**Compute fODF metrics**//
+    fodf_sh_order                                   = 8
+    fodf_sh_basis                                   = "descoteaux07"
+    fodf_peaks_absolute_factor                      = 2.0
+    fodf_peaks_relative_threshold                   = 0.1
+    fodf_peaks_ventricle_max_fa                     = 0.1
+    fodf_peaks_ventricle_min_md                     = 0.003
+
+    //**PFT tracking**//
+    compute_pft                                     = true
```

Comment:
This is not used and I would like the name to be run_pft, run_lt, run_fodf, and run_dti and finally run_registration so we are giving all the options for people to use this subworkflow as its max.

Reply:
i feel like if we already have a module to concatenate tractogram we should have a run_ensemble and combine both if asked.

Reply:
Let's put that in an issue